config/java_memory_assistant.yml

```diff
@@ -0,0 +1,33 @@
+# Encoding: utf-8
+# Cloud Foundry Java Buildpack
+# Copyright 2013-2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+version: 0.1.0
+repository_root: https://raw.githubusercontent.com/SAP/java-memory-assistant/repository
```

Comment:
We generally watch maven repositories of any type (Maven Central, Artifactory, etc.) for released versions and then cache them.  The tooling already built for this, so it's easy to integrate.  If we want to do something more aggressive (building tags of the Git repo), it's not impossible, but it'll take some time to get done.